python/cudf/cudf/core/dataframe.py

```diff
@@ -781,9 +781,15 @@ def __init__(
                 )
         elif isinstance(data, ColumnAccessor):
             raise TypeError(
-                "Use cudf.Series._from_data for constructing a Series from "
+                "Use cudf.DataFrame._from_data for constructing a Series from "
```

Comment:
Should that be `DataFrame`?
```suggestion
                "Use cudf.DataFrame._from_data for constructing a DataFrame from "
```

Reply:
Yup, thanks for the catch!